packages/jsonld/README.md

```diff
@@ -148,6 +150,18 @@ Performs JSON-LD flattening.
 Performs JSON-LD framing.
 
 **Returns**: <code>Promise.&lt;object&gt;</code> - Promise that resolves to the framed output.  
+<a name="module_jsonld..findNodeById"></a>
+
+### *jsonld*~findNodeById(document, id)
+
+| Param | Description |
```

Comment:
Should be "Parameter".

Reply:
This table is generated by the `jsdoc2md` with the default style and `iota.js` is using the same [style](https://github.com/iotaledger/iota.js/tree/next/packages/core#corecomposeapisettings).
I think it's okay to keep this table style.
If it still looks weird, I'll send another pull request to customize the table style.